src/ngraph/ngraph_emitter.cc

```diff
@@ -1441,6 +1445,28 @@ void Emitter::CreateLossOps() {
 
     return gradient;
   };
+  loss_op_backward_funcs_["MakeLoss"] = [this](const NodePtr& node,
+                                               const NgraphNodePtr& adjoint) {
+    auto input = op_map_[node->inputs_[0]];
+    const std::string norm =
+        get_default(node, "normalization", std::string("null"));
+    auto grad_scale =
+        makeConstant(node, get_default(node, "grad_scale", std::string("1.0")));
+
+    NgraphNodePtr grad;
+    if (norm == "valid") {
+      throw std::runtime_error(std::string("NGRAPH_BRIDGE: MakeLoss ") +
+                               "normalization not yet tested " +
+                               "in NGraph, please test with this script.");
+    } else if (norm == "batch") {
+      grad = grad_scale /
+             makeConstant(node, std::to_string(input->get_shape()[0]));
```

Comment:
no, but the implementation mirrors mxnet OP; Since the formula is straightforward, i assume it is same as mxnet op (which doesnt seem to have tests).

we could add a test under ngraph py unittest, by testing with mxnet op and comparing to its output.

Reply:
Hmm, that might be good to do for both valid and batch.